setup_monitoring.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+######################################################################################
+## Install Prometheus operator
+######################################################################################
+function install_prometheus_operator() {
+  echo "📦 installing the Prometheus operator..."
+  export KUBECONFIG=$KUBECONFIG_HOST
+
+  oc process -f config/monitoring/prometheus.tmpl.yaml \
+    -p NAMESPACE="$TOOLCHAIN_OPERATOR_NS_HOST" \
+    -p SA_NAME=prometheus-k8s \
+    | oc apply -f -
+  
+  # needed by the `kube-rbac-proxy` sidecar to create TokenReviews and SubjectAccessReviews
+  oc adm policy add-cluster-role-to-user kube-rbac-proxy -z prometheus-k8s -n "$TOOLCHAIN_OPERATOR_NS_HOST"
+  
+  echo "✅ done installing Prometheus operator"
+  echo ""
+}
+
+######################################################################################
+## Deploying Grafana
+######################################################################################
+function deploy_grafana() {
+  echo "📦 deploying Grafana..."
+  # fetch the route to Thanos Querier on member cluster and 
+  export KUBECONFIG=$KUBECONFIG_MEMBER
+  THANOS_QUERIER_URL_MEMBER="https://$(oc get route/thanos-querier -n openshift-monitoring -o json | jq -r '.status.ingress[0].host')"
+  export THANOS_QUERIER_URL_MEMBER
```

Comment:
why do we need to export this variable? the same applies also in the case of the other locally used vars...

Reply:
ahhh, that's because I did not use `local`! 🤦‍♂️